src/utilities/service-utils.test.ts

```diff
@@ -3,6 +3,7 @@ import { StubResourceRecord } from "../tests/stubs/stub-resource-record";
 import { Factory } from "rosie";
 import { AxiosResponse } from "axios";
 import { FactoryType } from "../tests/factories/factory-type";
+import { FactoryType as AndcultureCodeFactoryType } from "andculturecode-javascript-testing";
```

Comment:
I am not seeing where `FactoryType` or * being export  from "andculturecode-javascript-testing", only `export { AxiosResponseFactory } from "andculturecode-javascript-testing";` which is not being called. Think I am missing something

Reply:
It looks like it is being exported from `andculturecode-javascript-testing` here, @nhinhdao 

https://github.com/AndcultureCode/AndcultureCode.JavaScript.Testing/blob/43b5c171efc17beec852ab9343e1f4760e25e936/src/index.ts#L6

Reply:
I see. Thanks, @brandongregoryscott